frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -323,13 +323,47 @@ function ReaderRolling:addToMainMenu(menu_items)
                 end
             end
         },
+        {
+            text_func = function()
+                return T(_("Number of lines: %1"), G_reader_settings:readSetting("copt_overlap_lines") or 1)
+            end,
+            enabled_func = function()
+                return self.show_overlap_enable
+            end,
+            callback = function(touchmenu_instance)
+                local SpinWidget = require("ui/widget/spinwidget")
+                UIManager:show(SpinWidget:new{
+                    width = Screen:getWidth() * 0.75,
+                    value = G_reader_settings:readSetting("copt_overlap_lines") or 1,
+                    value_min = 1,
+                    value_max = 10,
+                    precision = "%d",
+                    ok_text = _("Set"),
+                    title_text =  _("Set overlapped lines"),
+                    text = _([[
+When page overlap is enabled, some lines from the previous pages are shown on the next page.
```

Comment:
```suggestion
When page overlap is enabled, some lines from the previous pages are displayed on the next page.
```
Just avoiding "shown" twice in a row. ;-)